common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -456,6 +465,52 @@ function (VideoPlayer, i18n) {
         });
     }
 
+    function loadYoutubePlayer() {
+        console.log(
+            '[Video info]: Start player in YouTube mode.'
+        );
+
+        this.fetchMetadata();
+        this.parseSpeed();
+    }
+
+    function loadHtmlPlayer() {
+        console.log(
+            '[Video info]: YouTube returned an error for ' +
+            'video with id "' + this.id + '".'
+        );
+
+        // When the youtube link doesn't work for any reason
+        // (for example, firewall) any
+        // alternate sources should automatically play.
+        if (!_prepareHTML5Video(this)) {
+            console.log(
+                '[Video info]: Continue loading ' +
+                'YouTube video.'
+            );
+
+            // Non-YouTube sources were not found either.
+
+            this.el.find('.video-player div')
+                .removeClass('hidden');
```

Comment:
I believe that ``is-hidden`` is the recommended class from the UX team now. @marcotuts, is that correct?